.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: release
```

Comment:
Should this PR remove the deploy from CircleCI? So that the next release does not have two systems pushing it.

Reply:
Everything CircleCI should be removed. I can certainly add the removal of the circleci script if we would like in this PR.